mk/travis-install-kcov.sh

```diff
@@ -24,9 +24,9 @@ set -ex
 
 # kcov 26 or newer is needed when getting coverage information for Rust.
 # kcov 31 is needed so `kcov --version` doesn't exit with status 1.
-KCOV_VERSION=${KCOV_VERSION:-31}
+KCOV_VERSION=${KCOV_VERSION:-36}
```

Comment:
The comments above this line explain why kcov 31 is needed. Please document why kcov 36 is needed.

Reply:
That should have been in the macos commit. I'll move it and add the comment.

Reply:
Comment added.